app/src/hooks/use-keyboard-shortcut-handler.ts

```
@@ -0,0 +1,50 @@
+        // Open help modal or navigate to help page
+        const helpButton = document.querySelector<HTMLButtonElement>('[data-testid="help-button"]');
+        if (helpButton) {
+          helpButton.click();
+        } else {
+          // Fallback to help page if no help button found
+          navigate({ to: "/help" });
+        }
+        break;
+      case "escape":
+        // Close modals, blur inputs, etc.
+        const activeElement = document.activeElement as HTMLElement;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)